plugins/Overlay/javascripts/Piwik_Overlay.js

```diff
@@ -47,7 +47,6 @@ var Piwik_Overlay = (function () {
             params.segment = segment;
         }
 
-        globalAjaxQueue.abort();
```

Comment:
Guess this one isn't needed, as the javascript actually does never push anything to the queue, so there's nothing that can be aborted, right?

Reply:
It's needed now because we're using AjaxHelper instead of piwikApi for one request. AjaxHelper pushes them onto the queue, but piwikApi didn't. So now this can result in a race condition where some requests don't display.